### PR TITLE
Set MIX_ENV during Phoenix digest

### PR DESCRIPTION
**Pull request by rjanja**
Corrects the current behavior of digest task not succeeding due to an assumed dev environment.

### CODE REVIEW

File: lib/bootleg/tasks/phoenix_digest.ex [hunk 1]
```diff
@@ -1,13 +1,14 @@
 defmodule Bootleg.Tasks.PhoenixDigest do
-  @moduledoc "Installs needed depedencies and calls `mix phoenix.digest`."
-  alias Bootleg.UI
+  @moduledoc "Installs dependencies and calls `mix phoenix.digest`."
+  alias Bootleg.{Config, UI}
 
   use Bootleg.Task do
     task :phoenix_digest do
+      mix_env = Config.get_config(:mix_env, "prod")
```
Comment (holetse) [inline lib/bootleg/tasks/phoenix_digest.ex:7]: It would be better to use the documented DSL verb `config`, and not an undocumented internal function here.

**Review by holetse — CHANGES_REQUESTED** [review]

**Comment by holetse** [comment]
This PR needs a test associated with the changes.

File: lib/bootleg/tasks/phoenix_digest.ex (hunk 1 above)
Comment (rjanja) [reply lib/bootleg/tasks/phoenix_digest.ex:7]: The DSL verb you're advocating is not documented nor is it used by any of Bootleg's own tasks, so I strongly disagree with you here.

**Comment by rjanja** [comment]
As it stands, this package doesn't work for a vanilla Phoenix 1.2 project. The test you are suggesting already exists, it's just not actually throwing an error and failing for reasons I've pointed out in #2 and again in #5. That fix has been separated into another PR at your request. At this point I'm feeling a little discouraged about my contributions back to the project, so maybe it'd be best for me to maintain a fork that works for my purposes and file upstream issues as I encounter them.

**Pull request closed without merging**